Assets/Scripts/DemoQuadTree.gd

```diff
@@ -51,7 +52,7 @@ func _ready() -> void:
 	root_qt_node.draw(0.2)
 	
 
-func _physics_process(delta: float) -> void:
+func _process(delta: float) -> void:
```

Comment:
Well, why did you change the _physics_process, tho?

Reply:
I can change it back if you like. The code didn't seem to have much to do with physics, so I had changed it when trying to work out what was going on.

Reply:
Ready to be merged.